Documentation/In-depth/TheInstallTool/Index.rst

```diff
@@ -99,8 +99,22 @@ administrators. As administrators still have elevated permissions you should
 still be careful which users can be trusted with these rights.
 
 At **Configuration Presets** TYPO3 offers presets for common settings groups.
-It allows for easier configuration of settings for debugging, image handling,
+It allows for easier configuration of settings for caching, debugging, image handling,
```

Comment:
It allows easier configuration ...

Reply:
Technically _allow for_ is correct here: https://www.merriam-webster.com/dictionary/allow%20for